src/app/utils/item.js

```diff
@@ -227,14 +238,19 @@ function LibraryItem() {
   };
 
   /**
-   * getHoldingLocation(item)
+   * getHoldingLocation(item, nonNyplRecap)
    * Returns updated location data from the holdingLocation property in the API for each item.
    * @param {object} item
+   * @param {boolean} nonNyplRecap
    * @return {object}
    */
-  this.getHoldingLocation = (item) => {
+  this.getHoldingLocation = (item, nonNyplRecap) => {
     let location = this.getDefaultLocation();
 
+    if (nonNyplRecap) {
```

Comment:
This is a good idea; best to be safe. It's nearly impossible to create a Sierra record without a location code (system default code = "zzzzz"), but an error could happen in indexing/ingest. I'll revise my review.